src/KDSoapClient/KDSoapClientInterface.cpp

```diff
@@ -332,7 +352,27 @@ void KDSoapClientInterface::setTimeout(int msecs)
     d->m_timeout = msecs;
 }
 
-#ifndef QT_NO_SSL
+bool KDSoapClientInterface::sendSOAP_ActionInHTTP_Header() const
```

Comment:
I don't want to nitpick too much, but I have to say the underscores after acronyms surprise me very much. My brain splits at underscores first, and then reads the 3 substrings, it doesn't really work.

Qt uses another solution: treating acronyms like normal words, i.e. with titlecase (for example the Http in "QNetworkProxy::HttpCachingProxy"). This would give

    sendSoapActionInHttpHeader()

Isn't this much more readable?
And consistent with Qt, which is an important point for a Qt-based library.

Reply:
> I don't want to nitpick too much
Hah, nitpicking is never too much.

>  And consistent with Qt, which is an important point for a Qt-based library.
I was not aware of this Qt coding style, but I will keep myself to this practice in the future.